runtime/plugin/health.vim

```diff
@@ -1 +1,7 @@
-command! -nargs=* CheckHealth call health#check([<f-args>])
+function! s:complete(lead, _line, _pos) abort
+  return filter(map(glob($VIMRUNTIME.'/autoload/health/*', 1, 1),
```

Comment:
Is this simple approach of concatenation good enough? I always forget the best way to do that for both, Windows and non-Windows.